### ISSUE TITLE
Interactive elements in mode-context-filter don’t receive keyboard focus

### ISSUE THREAD

**Issue by siggrid** [ushahidi/platform#4153]
**Describe the bug**

Element, that shows/hides dropdown doesn’t receive focus;
Show only and Hide controls are skipped when navigating inside dropdown.

**To Reproduce**

- Login as admin
- Go to Map page
- Use Tab key to navigate through the filters area

**Screenshots**

Skipped controls inside dropdown
![dropdown](https://user-images.githubusercontent.com/52042098/97924440-554f5980-1d68-11eb-8deb-881116219fe8.gif)

**Expected behavior**
Keyboard users can reach and operate interactive elements on the page

**Additional context**

It is also impossible to understand whether “Create new survey” link is focused or not.
<img width="305" alt="Screenshot 2020-11-02 at 22 47 08" src="https://user-images.githubusercontent.com/52042098/97924469-5f715800-1d68-11eb-815c-c07bc4a8a78f.png">

Aha! Link: https://ushahiditeam.aha.io/features/PROD-897
